### PR TITLE
Add series event

### PR DESCRIPTION
**Pull request by hanahmily**
Introduce the series event in order to help liaison to composite `seriesID`. This event contains filed names that indicates which fields should be included for building the `seriesID`

Signed-off-by: Gao Hongtao <hanahmily@gmail.com>

### CODE REVIEW

**Review by Fine0830 — APPROVED** [review]

File: api/event/discovery.go [hunk 1]
```diff
@@ -32,13 +32,22 @@ var (
 		ID:   ShardEventKindVersion.String(),
 		Type: bus.ChTypeUnidirectional,
 	}
+	SeriesEventKindVersion = common.KindVersion{
+		Version: "v1",
+		Kind:    "event-series",
+	}
+	TopicSeriesEvent = bus.Topic{
```
Comment (lujiajing1126) [inline api/event/discovery.go:39]: Since we have `bus.Uni` method, we should use them

**Comment by lujiajing1126** [comment]
LGTM.

As I understand, these events are defined for write, not query/read, right?

File: api/event/discovery.go (hunk 1 above)
Comment (hanahmily) [reply api/event/discovery.go:39]: make sense

**Comment by hanahmily** [comment]
> LGTM.
> 
> As I understand, these events are defined for write, not query/read, right?

yep, for the query scenario, the liaison will choose an arbitrary shard to send the request.

**Pull request merged**